service/history/historyEngine.go

```diff
@@ -3046,7 +3046,7 @@ func getWorkflowAlreadyStartedError(errMsg string, createRequestID string, workf
 	return &types.WorkflowExecutionAlreadyStartedError{
 		Message:        common.StringPtr(fmt.Sprintf(errMsg, workflowID, runID)),
 		StartRequestID: common.StringPtr(fmt.Sprintf("%v", createRequestID)),
-		RunID:          common.StringPtr(fmt.Sprintf("%v", runID)),
+		RunID:          fmt.Sprintf("%v", runID),
```

Comment:
weird.  why not just runID?  (not relevant to this change, so leave it if you want, just odd)

Reply:
Yep, fixed it.